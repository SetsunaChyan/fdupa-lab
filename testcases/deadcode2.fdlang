function main(){
x = 255;

while (x <= 255) {
    x = x - 1;
    check_interval(x, 0, 254);
}

check_interval(x, 0, 255);
}