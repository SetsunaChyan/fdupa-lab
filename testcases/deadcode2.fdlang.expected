Line 6: YES; x in [0, 254]
Line 9: Unreachable
